client/src/main.rs

```diff
@@ -585,6 +585,13 @@ fn fetch(
     let (State { peers, cidrs }, server_is_reachable) = match api.http("GET", "/user/state") {
         Ok(state) => (state, true),
         Err(ureq::Error::Transport(_)) => {
+            if store.peers().is_empty() {
+                bail!(
+                    "Could not connect to the innernet server and there are no cached peers, \
+                     bailing."
```

Comment:
I think the idiom is
```suggestion
                     bailing out."
```
...i.e. that it has a different meaning without the "out"?

Reply:
This app seems to use "exiting" in many other places, so I'll stick with that.